autoload/tagbar/types/uctags.vim

```diff
@@ -1332,6 +1333,34 @@ function! tagbar#types#uctags#init(supported_types) abort
         \ {'short' : 't', 'long' : 'tasks',               'fold' : 0, 'stl' : 1}
     \ ]
     let types.verilog = type_verilog
+    " SystemVerilog {{{1
+    let type_systemverilog = tagbar#prototypes#typeinfo#new()
+    let type_systemverilog.ctagstype = 'systemverilog'
+    let type_systemverilog.kinds     = [
+        \ {'short' : 'c', 'long' : 'constants',           'fold' : 0, 'stl' : 0},
+        \ {'short' : 'O', 'long' : 'constraints',         'fold' : 0, 'stl' : 1},
+        \ {'short' : 'Q', 'long' : 'prototypes',          'fold' : 0, 'stl' : 1},
+        \ {'short' : 'C', 'long' : 'classess',            'fold' : 0, 'stl' : 1},
+        \ {'short' : 'e', 'long' : 'events',              'fold' : 0, 'stl' : 1},
+        \ {'short' : 'E', 'long' : 'enumerators',         'fold' : 0, 'stl' : 1},
+        \ {'short' : 'f', 'long' : 'functions',           'fold' : 0, 'stl' : 1},
+        \ {'short' : 'm', 'long' : 'modules',             'fold' : 0, 'stl' : 1},
+        \ {'short' : 'P', 'long' : 'packages',            'fold' : 0, 'stl' : 1},
+        \ {'short' : 'b', 'long' : 'blocks',              'fold' : 0, 'stl' : 1},
+        \ {'short' : 'n', 'long' : 'net data types',      'fold' : 0, 'stl' : 1},
+        \ {'short' : 'R', 'long' : 'properties',          'fold' : 0, 'stl' : 1},
+        \ {'short' : 'S', 'long' : 'tructs',              'fold' : 0, 'stl' : 1},
+        \ {'short' : 'T', 'long' : 'typedefs',            'fold' : 0, 'stl' : 1},
+        \ {'short' : 'L', 'long' : 'clocking',            'fold' : 0, 'stl' : 1},
+        \ {'short' : 'V', 'long' : 'covergroup',          'fold' : 0, 'stl' : 1},
+        \ {'short' : 'b', 'long' : 'blocks',              'fold' : 0, 'stl' : 1},
+        \ {'short' : 'd', 'long' : 'defines',             'fold' : 0, 'stl' : 1},
+        \ {'short' : 'P', 'long' : 'programs',            'fold' : 0, 'stl' : 1},
+        \ {'short' : 'p', 'long' : 'ports',               'fold' : 0, 'stl' : 1},
+        \ {'short' : 'r', 'long' : 'register data types', 'fold' : 0, 'stl' : 1},
+        \ {'short' : 't', 'long' : 'tasks',               'fold' : 0, 'stl' : 1}
```

Comment:
This doesn't look fully complete, and the `blocks` kind is listed twice.

From the latest universal ctags, we have the following. Do we specifically want to exclude the missing ones? That is ok if we do, I just want to confirm as I'm not familiar to working with systemverilog or verilog:
```
host ~$ ctags --list-kinds=systemverilog | sort
A  assertions (assert, assume, cover, restrict)
b  blocks (begin, fork)
C  classes
c  constants (parameter, specparam, enum values)
d  text macros
E  enumerators
e  events
f  functions
H  checkers
i  instances of module or interface
I  interfaces
K  packages
L  clocking
l  interface class
M  modports
m  modules
n  net data types
N  nettype declarations
O  constraints
p  ports
P  programs
Q  prototypes (extern, pure) [off]
q  sequences
R  properties
r  variable data types
S  structs and unions
t  tasks
T  type declarations
V  covergroups
w  struct and union members
```